docs/cpp/config-wsl.md

```diff
@@ -49,20 +49,24 @@ To successfully complete this tutorial, you must do the following steps:
 
 1. Although you will be using VS Code to edit your code on Windows, you'll be compiling the code on Linux using the g++ compiler. You'll also debug on Linux using GDB. These tools are not installed by default on Ubuntu, so you have to install them. Fortunately, that task is quite easy!
 
-   From the command prompt, install the GNU compiler tools and the GDB debugger by typing:
+1. From the command prompt, first run apt-get update and dist-upgrade to bring the distro up to date. An out-of-date distro can sometimes interfere with attempts to install new packages.
 
    ```bash
-   sudo apt-get install build-essential gdb
+   sudo apt-get update && sudo apt-get dist-upgrade
    ```
-
-1. Verify that the install succeeded by navigating to the /user/bin directory and locating g++ and gdb:
+   
+1. From the command prompt, install the GNU compiler tools and the GDB debugger by typing:
 
    ```bash
-   cd $HOME
-   cd ../../usr/bin
-   ls g++ gdb
+   sudo apt-get install build-essential gdb
    ```
+1. Verify that the install succeeded by navigating to the /user/bin directory and locating g++ and gdb. If the filenames are not returned from the following command, try running the update and upgrade command again.
 
+```bash
```

Comment:
Can you indent this code block 3 spaces? Otherwise it won't be indented correctly under the step text.

Reply:
I can do it to get this out today.

Reply:
@gregvanl thanks I missed that. Done.

Reply:
ok never mind i see you took care of it. thanks!